docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -719,6 +721,30 @@ testcafe ie my-tests --page-load-timeout 0
 
 *Related configuration file property*: [pageLoadTimeout](configuration-file.md#pageloadtimeout).
 
+### --ajax-request-timeout \<ms\>
+
+Specifies wait time (in milliseconds) for fetch/XHR requests. If no response is received within the specified period, an error is thrown.
+
+```sh
+testcafe chrome my-tests --ajax-request-timeout 40000
+```
+
+**Default value**: `120000`
+
+*Related configuration file property*: [ajaxRequestTimeout](configuration-file.md#ajaxrequesttimeout)
+
+### --page-request-timeout \<ms\>
+
+Specifies time (in milliseconds) to wait for a server to serve an HTML page. If the page isn't served within the specified period, an error is thrown.
```

Comment:
Both XHR requests and HTML pages responded by http server.
So, we can simplify description for the `--page-request-timeout` option in the same way as `--ajax-request-timeout`.

`Specifies wait time (in milliseconds) for HTML pages. If the page isn't received within the specified period, an error is thrown.`